hr_employee_product/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015-17 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+
+{
+    'name': "Manage products representing employees",
+    'version': '12.0.1.0.0',
+    'category': 'Human Resources',
+    'author': "Eficent, Odoo Community Association (OCA)",
+    'website': 'http://www.github.com/OCA/hr-timesheet',
+    'license': 'AGPL-3',
+    'summary': 'Product is an employee',
+    "depends": ['product', 'hr_timesheet'],
+    "data": [
```

Comment:
`data` sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
Hi @nikul-serpentcs, i have updated the code with those changes. Could you update your review please?